newrelic/resource_newrelic_cloud_aws_integrations.go

```diff
@@ -0,0 +1,710 @@
+package newrelic
+
+import (
+	"context"
+	"github.com/newrelic/newrelic-client-go/pkg/cloud"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceNewRelicCloudAwsIntegrations() *schema.Resource {
```

Comment:
I've just finished a full end to end test including the push integration, and .. it works 🥳  

Great work @mbazhlekova